eth2util/signing/signing.go

```diff
@@ -29,10 +29,11 @@ import (
 type DomainName string
 
 const (
-	DomainBeaconProposer DomainName = "DOMAIN_BEACON_PROPOSER"
-	DomainBeaconAttester DomainName = "DOMAIN_BEACON_ATTESTER"
-	DomainRandao         DomainName = "DOMAIN_RANDAO"
-	DomainExit           DomainName = "DOMAIN_VOLUNTARY_EXIT"
+	DomainBeaconProposer     DomainName = "DOMAIN_BEACON_PROPOSER"
+	DomainBeaconAttester     DomainName = "DOMAIN_BEACON_ATTESTER"
+	DomainRandao             DomainName = "DOMAIN_RANDAO"
+	DomainExit               DomainName = "DOMAIN_VOLUNTARY_EXIT"
+	DomainApplicationBuilder DomainName = "DOMAIN_APPLICATION_BUILDER"
```

Comment:
This requires an additional update in go-eth2-client

PR has been raised waiting for Jim to review

https://github.com/attestantio/go-eth2-client/pull/23